sekizai/templatetags/sekizai_tags.py

```diff
@@ -82,6 +94,8 @@ def render_tag(self, context, name, postprocessor, nodelist):
         if postprocessor:
             func = import_processor(postprocessor)
             data = func(context, data, name)
+        elif postprocessor is None and name in _mapped_postprocessors:
```

Comment:
why check ``postprocessor`` for ``None``? The ``if`` clause checks if there is  a``postprocessor`` so the ``else`` doesn't need to check that again.